robottelo/common/helpers.py

```diff
@@ -70,6 +70,25 @@ def get_server_url():
         ))
 
 
+def get_internal_docker_url():
+    """Returns the docker internal server url config
+
+    If found the variable ``{server_hostname}`` in the string, it will be
+    replaced by main.server.hostname config
```

Comment:
If the variable ``{server_hostname}`` is found in the string, it will be
    replaced by ``main.server.hostname`` from the config file.